presto-hive/src/main/java/com/facebook/presto/hive/PrismTransactionHandle.java

```diff
@@ -0,0 +1,17 @@
+package com.facebook.presto.hive;
+
+public class PrismTransactionHandle
```

Comment:
We can't expose prism concept to open source. Instead, override `HiveTransactionManager` internally.